editor/store/effects.js

```diff
@@ -406,6 +418,11 @@ export default {
 			optimist: { type: BEGIN, id: transactionId },
 		} );
 
+		// TODO: these are potentially undefined, this fix is in place
+		// until there is a filter to not use reusable blocks if undefined
+		if ( ! has( wp, 'api.models.Blocks' ) ) {
+			return;
+		}
```

Comment:
Let's move this check to the very top of the function. Right now if `wp.api.models.Blocks` is undefined then we will dispatch `REMOVE_REUSABLE_BLOCK` without ever telling `redux-optimist` to commit it.

Reply:
:+1: